crates/logic/src/server/network/login.rs

```diff
@@ -18,20 +17,20 @@ impl FalconServer {
         let player_uuid = Uuid::new_v3(&Uuid::NAMESPACE_DNS, username.as_bytes());
         let username2 = username.clone();
         connection.execute_sync(move |connection| {
-            // TODO: remove ignore
             connection
-                .send_packet(LoginSuccessSpec::new(player_uuid, username2), falcon_send::write_login_success)
-                .ignore();
+                .send_packet(LoginSuccessSpec::new(player_uuid, username2), falcon_send::write_login_success)?;
             let handler_state = connection.handler_state_mut();
             handler_state.set_connection_state(ConnectionState::Play);
             handler_state.set_player_uuid(player_uuid);
+
+            Ok(())
```

Comment:
Either add an empty line after the `send_packet` call or remove it here.
```suggestion
            Ok(())
```